src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

```diff
@@ -65,89 +67,93 @@ import dashboardsTimePicker from 'images/dashboards_screenshot-crop_time-picker.
 
-Once you have customized your dashboard and built your charts, use our advanced visualization features and tools for data exploration to correlate and analyze your data.
+Once you've customized your dashboard and built your charts, use our advanced visualization features and tools for data exploration to correlate and analyze your data.
 
 ## Customize your dashboard [#dash-manage]
 
 Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the dashboards page.
 
-<Callout variant="tip">
-  Click the <Icon name="fe-tag"/> icon to access the **See metadata and manage tags** modal. There you can see the dashboard's entity GUID, account ID, account name, and manage all the tags added to the dashboard.
-</Callout>
-
 <img
-  width="50%;"
-  title="Dashboards: See metadata and manage tags"
-  alt="Dashboards: See metadata and manage tags"
-  src={dashboardsDashboardMetadata}
+  title="Dashboards index"
+  alt="Dashboards index"
+  src={dashboardsDashboardIndex}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards**: See metadata and manage tags.
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index.
 </figcaption>
 
-This page includes these features:
+Click a dashboard to open its charts. A dashboard page includes these features:
 
 <CollapserGroup>
+<Collapser
+    id="dash-clone"
+    title="Duplicate your dashboard"
+  >
+    You can duplicate your dashboard regardless of your permission levels. Click the <Icon name="fe-more-horizontal"/> icon on the right corner and select **Duplicate dashboard**.
+
+    The duplicated dashboard is named like the original dashboard followed by the word “copy”. For example, if you duplicate a dashboard named `This is my dashboard`, the duplicate is called `This is my dashboard copy`. You can change the name, select the associated account, and add the permissions when you duplicate it. Once you click **Create**, you'll see a pop-up message on your screen to access the duplicated dashboard. You can edit the name and other properties of the dashboard, such as permissions, any time.
+    
+    The dashboard is automatically copied and the duplicate is added to the [index](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#get-started). The duplicate has [**Public - Read and write** permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions). 
+    
+  </Collapser>
 <Collapser
     id="dash-edit"
     title="Edit your dashboard"
   >
-    You can edit a dashboard if you have [permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions) to do it.
+    You can edit a dashboard if you've [permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions) to do it.
```

Comment:
I could be the odd one out here, but I feel like "you have" if one of the few contractions we should refrain from using. It feels a little awkward to me. [Here's](https://www.oxfordlearnersdictionaries.com/us/definition/english/you-ve#:~:text=%E2%80%8Byou%20have%20You've,Questions%20about%20grammar%20and%20vocabulary%3F ) some food for thought.

Reply:
Done!